examples/imdb_cnn_lstm.py

```diff
@@ -0,0 +1,86 @@
+from __future__ import absolute_import
+from __future__ import print_function
+import numpy as np
+np.random.seed(1337)  # for reproducibility
+
+from keras.preprocessing import sequence
+from keras.optimizers import SGD, RMSprop, Adagrad
+from keras.utils import np_utils
+from keras.models import Sequential
+from keras.layers.core import Dense, Dropout, Activation
+from keras.layers.embeddings import Embedding
+from keras.layers.recurrent import LSTM, GRU, SimpleRNN
+from keras.layers.convolutional import Convolution1D, MaxPooling1D
+from keras.datasets import imdb
+
+'''
+    Train a recurrent convolutional network on the IMDB sentiment classification task.
+
+    GPU command:
+        THEANO_FLAGS=mode=FAST_RUN,device=gpu,floatX=float32 python imdb_lstm.py
+        
+    Get to 0.8498 test accuracy after 2 epochs. 41s/epoch on K520 GPU.
+'''
+
+
+#Parameters
+
+#Embedding
+max_features = 20000
+maxlen = 100 
+embedding_size = 128
+
+#Convolution
+filter_length = 3
+nb_filter = 64
+pool_length = 2
+
+#LSTM
+lstm_output_size = 70
+
+#Training
+batch_size = 30
+nb_epoch = 2
+'''
+Note:
+batch_size is highly sensitive.
+Only 2 epoches are done as the dataset is too small.
+'''
+
+
+print("Loading data...")
+(X_train, y_train), (X_test, y_test) = imdb.load_data(nb_words=max_features, test_split=0.2)
+print(len(X_train), 'train sequences')
+print(len(X_test), 'test sequences')
+
+print("Pad sequences (samples x time)")
+X_train = sequence.pad_sequences(X_train, maxlen=maxlen)
```

Comment:
@farizrahman4u Right, I see that, but if they are padded, doesn't there need to be Masking so that Embedding knows not to pass the zeros on? I tried to add that, but it seems that Convolution1D is not yet compatible with Masking. Do you know what the effect of the padded zeros is on training in this example? 

Thanks.